SPECS/sg3_utils/sg3_utils.spec

```diff
@@ -26,35 +26,41 @@ Package containing static library object for development.
 %setup -q
 
 %build
-#make some fixes required by glibc-2.28:
-sed -i '/unistd/a #include <sys/sysmacros.h>' src/sg_dd.c src/sg_map26.c src/sg_xcopy.c
-
+./autogen.sh
 %configure
 
+# Don't use rpath!
+sed -i 's|^hardcode_libdir_flag_spec=.*|hardcode_libdir_flag_spec=""|g' libtool
+sed -i 's|^runpath_var=LD_RUN_PATH|runpath_var=DIE_RPATH_DIE|g' libtool
+
 %install
-make DESTDIR=%{buildroot} install %{?_smp_mflags}
-install -m 755 scripts/scsi_logging_level %{buildroot}/%{_bindir}
-install -m 755 scripts/rescan-scsi-bus.sh %{buildroot}/%{_bindir}
+if [ "$RPM_BUILD_ROOT" != "/" ]; then
+        rm -rf $RPM_BUILD_ROOT
+fi
 
-%post -p /sbin/ldconfig
-%postun -p /sbin/ldconfig
+make install \
+        DESTDIR=$RPM_BUILD_ROOT
 
 %files
 %defattr(-,root,root)
-%license COPYING
-%{_bindir}/*
-%{_mandir}/*
+%doc AUTHORS ChangeLog COPYING COVERAGE CREDITS INSTALL NEWS README README.sg_start
+%attr(755,root,root) %{_bindir}/*
+%{_mandir}/man8/*
 
 %files -n libsg3_utils-devel
 %defattr(-,root,root)
-%{_libdir}/libsgutils2.a
+%{_includedir}/scsi/*.h
+%{_libdir}/*.so
+%{_libdir}/*.a
+%{_libdir}/libsgutils2-1.48.so.2
```

Comment:
can we use a macro here for the version, so this doesn't need to be manually changed for the next upgrade?

Reply:
something like `%{_libdir}/libsgutils2-%{version}.so.2`

Reply:
done